code/modules/bitrunning/objects/byteforge.dm

```diff
@@ -5,25 +5,57 @@
 	desc = "A machine used by the quantum server. Quantum code converges here, materializing decrypted assets from the virtual abyss."
 	icon = 'icons/obj/machines/bitrunning.dmi'
 	icon_state = "byteforge"
+	base_icon_state = "byteforge"
 	obj_flags = BLOCKS_CONSTRUCTION | CAN_BE_HIT
 	/// Idle particles
 	var/mutable_appearance/byteforge_particles
 
 /obj/machinery/byteforge/Initialize(mapload)
 	. = ..()
 
-	return INITIALIZE_HINT_LATELOAD
```

Comment:
You forgot to re-add this

Reply:
It isn't necessary if you look at the parent  initialize proc

https://github.com/tgstation/tgstation/blob/90783148b8386557a1970c4bca5fd4fc528953b0/code/game/machinery/_machinery.dm#L195

Actually we can remove this whole proc